spec/index.html

```diff
@@ -4253,21 +4254,21 @@ <h4>Templates with Blank Nodes</h4>
                 <pre class="resultGraph">
 @prefix vcard: &lt;http://www.w3.org/2001/vcard-rdf/3.0#&gt; .
 
-_:v1 vcard:N         _:x .
-_:x vcard:givenName  "Alice" .
-_:x vcard:familyName "Hacker" .
+_:a vcard:N         _:v1 .
+_:v1 vcard:givenName  "Alice" .
+_:v1 vcard:familyName "Hacker" .
 
-_:v2 vcard:N         _:z .
-_:z vcard:givenName  "Bob" .
-_:z vcard:familyName "Hacker" .
+_:b vcard:N         _:v2 .
+_:v2 vcard:givenName  "Bob" .
+_:v2 vcard:familyName "Hacker" .
                 </pre>
               </div>
             </div>
           </div>
-          <p>The use of variable <code>x</code> in the template, which in this example will be bound
-            to blank nodes with labels <code>_:a</code> and <code>_:b</code> in the data, causes
-            different blank node labels (<code>_:v1</code> and <code>_:v2</code>) in the resulting RDF
-            graph.</p>
+          <p>The use of the blank node <code>_:v</code> in the template will in this example be scoped to the two query solutions.
+            This will cause it to be bound to blank nodes with labels <code>_:v1</code> and <code>_:v2</code>.</p>
+          <p>As the blank node labels <code>_:a</code> and <code>_:b</code> originate from query solutions
+            from the underlying RDF dataset, they will not be rebound.</p>
```

Comment:
Saying blank node identifier are "bound" makes them variable-like. They are "replaced" ("substituted"?) - it's template.

```suggestion
          <p>
            The blank node with identifier <code>_:v</code> in the template
            will be replaced by a different blank node when the template is applied
            to each of the two query solutions.
            In this example, this will cause the template to generate blank nodes
            with identifiers <code>_:v1</code> and <code>_:v2</code> in the
            results graph.
          </p>
          <p>
            The blank nodes in the query solutions, shown with identifiers 
            <code>_:a</code> and <code>_:b</code>, originate from the underlying
            RDF dataset and will not be altered.
          </p>
```
(with reflowing - hopefully making it easier to read in review.)